apps/web/src/lib/components/home/Hero.svelte

```diff
@@ -1,25 +1,53 @@
-<section class="h-[80vh] bg-hero bg-cover bg-center bg-no-repeat px-4">
-	<div
-		class="container mx-auto flex h-full flex-col items-center justify-center"
-	>
-		<h1 class="h1 z-10 text-center text-5xl font-bold text-white">
-			Welcome to Freedom in Christ Church.
-		</h1>
-		<h2 class="h2 my-8 text-center text-2xl text-white md:w-[50rem]">
-			A Christ-centred, Holy Spirit-led community committed to the building of
-			strong relationships and planting of healthy reproducing churches.
-		</h2>
-		<div class="my-8 flex gap-8">
-			<a
-				href="/about"
-				class="bg-surface-200-700-token text-token btn btn-xl"
-				data-sveltekit-preload-data="hover">I'm new</a
+
+<section style="background-image: url('/images/youngPeopleImage.jpg');" class="flex h-[30vh] sm:h-[90vh] bg-contain bg-no-repeat sm:bg-cover sm:bg-gradient-to-t sm:from-blue-950 to-green-800 items-center justify-center ">
+	
+	<div style="background-image: url('/images/youthImage.jpg');" class="sm:rounded-2xl sm:bg-gradient-to-t sm:from-blue-950 to-green-800 mt-40 sm:mt-[480px] mx-auto bg-contain bg-no-repeat flex h-[420px] sm:h-auto p-4 items-center justify-around ">
```

Comment:
i like the `<section>` background, but i think adding another image on top of that is a little too much. We can probably remove this image and then style the rest of the text instead